src/preferences/dialog/dlgpreflibrary.h

```diff
@@ -22,7 +22,8 @@ class DlgPrefLibrary : public DlgPreferencePage, public Ui::DlgPrefLibraryDlg  {
     enum TrackDoubleClickAction {
         LOAD_TO_DECK,
         ADD_TO_AUTODJ_BOTTOM,
-        ADD_TO_AUTODJ_TOP
+        ADD_TO_AUTODJ_TOP,
+        DO_NOTHING
     };
```

Comment:
Can you make this an enum class with CamelCase members?

Also, I think `DO_NOTHING` should be renamed to `IGNORE` for consistency with the radio button.

Reply:
I can try it.
camelCase or PascalCase?

re: ignore see [@daschuer 's remark above](https://github.com/mixxxdj/mixxx/pull/3194#discussion_r509695739), the Windows CI failed earlier because of that.

Reply:
PascalCase, sorry.

If you use an enum class, there shouldn't be a naming conflict because enum classes as properly scoped (unlink oldschool enums) and the name will be different: `TrackDoubleClickAction::Ignore` vs. `IGNORE`.

Reply:
Done.

Reply:
I always wonder how you're understanding my review comments considering all the typos introduced by my phone's "aurocorrect" xD

Reply:
I have a built-in autocorrect filter ;|

Reply:
thinking about it... a buddy of mine has dyslexia, or heavy difficulties with writing,. I think he gave trying to do it correctly. so he's trained my phantasy and ability to abstract.. a lot!